lib/routes/collection.js

```diff
@@ -75,7 +75,8 @@ const routes = function (config) {
         throw new Error('Query entered is not valid');
       }
     }
-    return [
+    // queryOptions exists if is an aggregate
+    return queryOptions ? [
```

Comment:
Could you add a test or supertest for this change, so we ensure the returnwith queryOptions filled or empty in next development?

Reply:
@shakaran Added unit tests and added `json` extension.

FYI @BlackthornYugen

Reply:
I will add further unit tests for the aggregate query.